health/health.d/memory.conf

```diff
@@ -43,5 +50,9 @@ component: Memory
     every: 10s
      warn: $this > 0
     delay: down 1h multiplier 1.5 max 1h
-     info: amount of memory corrupted due to a hardware failure
+     info: Amount of memory corrupted due to a hardware failure. \
+           Memory is corrupted due to a hardware failure. \
+           While the error may be due to a failing DRAM chip, \
+           it can also be caused by incorrect seating or improper contact between the socket and DIMM. \
```

Comment:
And again same as both above, it may be a transient error due to cosmic rays.

Reply:
> It may also just be a transient error due to cosmic rays.

Nice to know, but I think it is ok to not mention it (unless NASA uses Netdata).